### PR TITLE
attempt to sort out licenses

### LINKED ISSUE #1: License
Thoughts on (c) Claire Nord, Noah Moroze, Kye Burchard, and Rinik Kumar? @kyeb @nmoroze @rk9109

### PR DESCRIPTION
**Pull request by cmnord**
Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -313,3 +313,13 @@ Follow along with the slides **[here](slides.pdf)**!
   * Microphone + LED visualizer
   * Bop-it
 * [SLIDE 13] fin.
+
+## Licenses
+
+This curriculum and `slides.pdf` are licensed under CC BY-SA 4.0 (see [LICENSE_CURRICULUM.txt](LICENSE_CURRICULUM.txt)).
+
+The example code under `1.3_blink/`, `1.5_fade/`, and `2.1_button` is in the public domain.
```
Comment (nmoroze) [inline README.md]: `2.1_button` -> `2.1_button/` ?

File: README.md [hunk 2]
```diff
@@ -313,3 +313,13 @@ Follow along with the slides **[here](slides.pdf)**!
...
+
+The example code under `2.2_photoresistor/`, `3.4_lcd_control/`, and `reaction_game` is under the [MIT license](reaction_game/LICENSE.txt).
```
Comment (nmoroze) [inline README.md]: same thing, `reaction_game` -> `reaction_game/`?

File: README.md [hunk 3]
```diff
@@ -313,3 +313,13 @@ Follow along with the slides **[here](slides.pdf)**!
...
+
+The example code under `1.6_rgb_led/` and `3.3_helloworld/` is (presumably?) (c) [Elegoo Inc.](https://www.elegoo.com/).
```
Comment (nmoroze) [inline README.md]: maybe could also say like "blah blah comes from Elegoo Arduino Starter Kit example code (insert more specific link), with no explicit license provided" or something?

Comment (cmnord) [reply README.md]: ah yeah, that seems better.

**Comment by kyeb** [comment]
iirc,  `reaction_game` was entirely Rinik, and should not have my name on it haha

File: README.md (hunk 1 above)
Comment (cmnord) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (cmnord) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (cmnord) [reply README.md]: done

**Comment by cmnord** [comment]
> iirc, `reaction_game` was entirely Rinik, and should not have my name on it haha

Fixed!

**Comment by cmnord** [comment]
will merge once @rk9109 takes a look

**Pull request merged**